hugo.toml

```diff
@@ -127,6 +127,7 @@ id = "UA-00000000-0"
 copyright_k8s = "The Kubernetes Authors"
 copyright_linux = "Copyright © 2020 The Linux Foundation ®."
 
+contribution_guidelines_url = "https://www.kubernetes.dev/docs/comms/"
```

Comment:
Either we change this link or override the Docsy's default hyperlink text [`community_guideline`](https://github.com/google/docsy/blob/ab1f1d0257a09cce2fd820665b1b4900bf38921c/i18n/en.toml#L79) in our [en.toml](https://github.com/kubernetes/website/blob/main/data/i18n/en/en.toml) to accurately reflect where the link leads.

 > // Docsy’s [English-language strings](https://github.com/google/docsy/blob/main/i18n/en.toml)
[community_guideline]
other = "Contribution Guidelines"

Reply:
I think what we really, really need are some actual contribution guidelines on https://k8s.dev/

“Implementing these is left as an exercise for the reader”

Reply:
Maybe https://www.kubernetes.dev/docs/contributor-cheatsheet/ is a better place to link to?

Reply:
> Maybe https://www.kubernetes.dev/docs/contributor-cheatsheet/ is a better place to link to?

Agree, this link (https://www.kubernetes.dev/docs/contributor-cheatsheet/) will be an improvement over the current one. Additionally, adding a comment to localize the string `community_guideline` would be helpful too.